src/v/kafka/tests/group_test.cc

```diff
@@ -47,11 +47,12 @@ static group get() {
     return group(kafka::group_id("g"), group_state::empty, conf, nullptr);
 }
 
-static const std::vector<member_protocol> test_protos = {
+static const std::vector<member_protocol> test_group_protos = {
```

Comment:
clashes with static globals in `member_test.cc` with the same names - could alternatively just exclude one of the 2 files with `SKIP_UNITY_BUILD_INCLUSION`